In [1]:
#Consider the string given below and answer the following questions
string = "OQYWFClFhFGAvIWYwGKpmZhnJiyzTslSIhSwvOsqJMEphzmifTkyqOMNpnOtXZxmCfgDYqbaBHAUvIWhMnvwZnEMVDvmEfLrDoQnAZgQEgXQVnmSYkfedpAdhrtpOgORpYLRZYGWdhWYuqQssCUXtTzKRDAhpjUheOzUroZNzWFtZOVwIapzUYtbSbjYNErzQ"
len(string)

193

####  Calculate the number of characters in the string

-  193
-  200
-  300
-  500


In [2]:
# Type your code here
c=0
for i in string:
  if(i=='a'):
    c+=1
print(c)

2


#### Count the number of occurences of 'a' in the string

-  1
-  2
-  10
-  5

In [11]:
string[:2]=="if"

False

#### Identify whether the string starts with 'if'.

-  True
-  False

In [12]:
# Type your code here
string[63:89]

'mCfgDYqbaBHAUvIWhMnvwZnEMV'

#### Return the substring starting at index 63 and ending at index 88  [Both the indexes are inclusive]

-  'OQYWFClFhFGA'
-  'mCfgDYqbaBHAUvIWhMnvwZnEMV'
-  'IWYwGKpmZhnJiyzTslSIhSwv'
-  'JMEphzmifTkyqOMNpnOtXZ'

In [13]:
# Type your code here
string[45]

'z'

#### Identify the character at index 45

-  a
-  b
-  z
-  x

In [ ]:
# Type your code here